irods_consortium_continuous_integration_test_hook.py

```diff
@@ -341,25 +321,36 @@ def main():
     parser = optparse.OptionParser()
     parser.add_option('--output_root_directory')
     parser.add_option('--built_packages_root_directory')
+    parser.add_option('--test', metavar='dotted name')
+    parser.add_option('--skip-setup', action='store_false', dest='do_setup', default=True)
     options, _ = parser.parse_args()
 
-    output_root_directory = options.output_root_directory
     built_packages_root_directory = options.built_packages_root_directory
     package_suffix = irods_python_ci_utilities.get_package_suffix()
     os_specific_directory = irods_python_ci_utilities.append_os_specific_directory(built_packages_root_directory)
 
-    irods_python_ci_utilities.install_os_packages_from_files(
-        glob.glob(os.path.join(os_specific_directory, 'irods-auth-plugin-krb*.{0}'.format(package_suffix))))
-    install_testing_dependencies()
+    if options.do_setup:
+        irods_python_ci_utilities.install_os_packages_from_files(
+            glob.glob(os.path.join(os_specific_directory,
+                      'irods-auth-plugin-krb*.{}'.format(package_suffix))
+            )
+        )
+
+        add_alias_to_etc_hosts()
+
+        install_testing_dependencies()
+
+        configure_kerberos_for_testing()
 
-    time.sleep(10)
+    test = options.test or 'test_irods_auth_plugin_krb'
```

Comment:
Consider renaming `test` (how about `test_name`)?

Feels a little loose.

Reply:
I've created a separate issue for this in the testing environment: https://github.com/irods/irods_testing_environment/issues/81